_rules/aria-attr-defined-5f99a7.md

```diff
@@ -6,9 +6,9 @@ description: |
   This rule checks that each `aria-` attribute specified is defined in ARIA 1.2.
 accessibility_requirements:
   wcag20:1.3.1: # Info and Relationships (A)
-    secondary: true
+    secondary: This success criterion is **less strict** than this rule. This is because the rule does not ignore irrelevant ARIA properties. This is why some of the failed examples may not satisfy this success criterion.
```

Comment:
This rule is **more strict** than the success criterion because it does not ignore irrelevant ARIA properties. This means some of the reported failures can be false positives rather than a success criterion failure.

Reply:
I like the change from saying "the rule more strict" (than the SC)  to "the SC is less strict" (than the rule).